annotation-client/src/main/java/eu/europeana/annotation/client/connection/AnnotationApiConnection.java

```diff
@@ -459,23 +473,30 @@ public AnnotationOperationResponse createEuropeanaAnnotationLd(String motivation
      * request:
      * http://localhost:8080/annotation/provider/identifier{.jsonld}?wskey=apidemo
      * where identifier is: http://data.europeana.eu/annotation/webanno/494
-     * 
-     * @param wskey
      * @param identifier The identifier URL that comprise annotation provider and ID
+     * @param user TODO
+     * @param wskey
      * @param userToken
      * @param format
+     * 
      * @return response entity that comprises response headers and status code.
      * @throws IOException
      */
-    public ResponseEntity<String> deleteAnnotation(String identifier) throws IOException {
+    public ResponseEntity<String> deleteAnnotation(String identifier, String user) throws IOException {
 
 	String url = getAnnotationServiceUri();
 	url += WebAnnotationFields.SLASH + identifier;
 
 	/**
 	 * Execute Europeana API request
 	 */
-	return deleteURL(url, authorizationHeaderName, regularUserAuthorizationValue);
+	if (user!=null && user.equals(USER_ADMIN)) {
```

Comment:
please replace as indicated in the previous comments

Reply:
Done.